modules/apps/commerce/commerce-frontend-taglib/src/main/java/com/liferay/commerce/frontend/taglib/servlet/taglib/PriceTag.java

```diff
@@ -148,6 +168,8 @@ public void setPageContext(PageContext pageContext) {
 
 		_productHelper = ServletContextUtil.getProductHelper();
 		_configurationProvider = ServletContextUtil.getConfigurationProvider();
+		_commerceChannelService =
+			ServletContextUtil.getCommerceChannelService();
```

Comment:
We should sort this by alphabetical order.

```suggestion
		_commerceChannelService =
			ServletContextUtil.getCommerceChannelService();
		_configurationProvider = ServletContextUtil.getConfigurationProvider();
		_productHelper = ServletContextUtil.getProductHelper();
```